src/index.js

```diff
@@ -1,18 +1,13 @@
-/** @jsx Preact.h */
 import Preact from 'preact';
-import ErrorBox from './error';
-import App from './components/app';
+import { getScrollytellers } from './loader';
 
-let sections = () => {
-    __ODYSSEY__.utils.anchors.getSections('scrollyteller').forEach(section => {
-        let element = section.betweenNodes.shift().firstChild;
-        render(element, section);
-        section.substituteWith(element);
-    });
+let init = () => {
+    getScrollytellers().forEach(section => render(section.mountNode, section));
```

Comment:
As discussed, we may need to do something here to provide a hook for any interactives which load after this has already run.